tests/unit/AnnualReport.spec.ts

```diff
@@ -2334,29 +2320,19 @@ describe('AnnualReport - BCOL error dialog on save', () => {
     vm.totalFee = 100
 
     // sanity check
-    expect(vm.bcolObj).toBeNull()
+    expect(vm.saveErrors).toStrictEqual([])
 
     const button = wrapper.find('#ar-file-pay-btn')
     expect(button.attributes('disabled')).toBeUndefined()
 
-    // Stub out a response from the Error endpoint in Pay API
-    get.withArgs('codes/errors/BCOL_ERROR')
```

Comment:
The filing views no longer call the Pay API to fetch the error content -- it's already embedded in the PUT or POST error response.